src/View/Input.js

```diff
@@ -0,0 +1,40 @@
+import { MissionUtils } from "@woowacourse/mission-utils";
+import { INPUT_MESSAGES } from "../constants/message.js";
+
+/** 입력과 관련된 로직들을 관리하는 class입니다. */
+class Input {
+  /** 로또 총 구매 금액을 입력받습니다. */
+  async inputMoney() {
+    try {
+      const input = await MissionUtils.Console.readLineAsync(INPUT_MESSAGES.PURCHASE_MONEY);
+      return input;
```

Comment:
```suggestion
      return MissionUtils.Console.readLineAsync(INPUT_MESSAGES.PURCHASE_MONEY);
```
이렇게 사용하는거랑 기존 코드랑은 어떤 차이가 있을까요??

Reply:
오우 차이가 없네요 ㅎㅎ 바꿔도 괜찮을 것 같아용 :)